glean_parser/translate.py

```diff
@@ -113,6 +147,9 @@ def translate_metrics(
     if parser_config.get("allow_reserved"):
         options["allow_reserved"] = True
 
+    # Show the outputter where to find the optional transformer
+    options["transform"] = transform_metrics
```

Comment:
Ah, sorry, I wasn't clear.  I think if we're going to do this, we might as well just have the outputter call the function directly.  (I think it's a little opaque that's it's passed in the `options` dict...)  That, of course, requires each of the outputters to call this (which now they don't do).

Alternatively (and maybe better because it localizes things) -- we could have a `transform_metrics` flag on the `Outputters` metadata [here](https://github.com/mozilla/glean_parser/blob/c9e4507b36bfe7e27412f2f51866ce1398860dff/glean_parser/translate.py#L44).  This would be `True` for everything except markdown.  Then at this line we would perform metric transformation (or not) based on that flag.

Reply:
I like the magic transform thing, but it'd be 100% False for the foreseeable future as I'm implementing `rate` just for FOG and FOG doesn't use `translate` at all.

So, come to think of it, that's a real problem with my design. I guess I'd have to use the exposed `translate.transform` by name _anyway_. Lemme rework to that.